iceoryx_posh/test/moduletests/test_roudi_portmanager.cpp

```diff
@@ -668,3 +714,46 @@ TEST_F(PortManager_test, PortDestroy)
         }
     }
 }
+
+TEST_F(PortManager_test, AcquireInterfacePortDataAfterDestroyingPreviouslyAcquiredOnesIsSuccessful)
+{
+    std::vector<iox::popo::InterfacePortData*> interfaceContainer;
+    std::string itf = "itf";
+
+    // first aquire all possible interfaces
+    exhaustInterfacePorts(itf, [&](auto InterafcePort) { interfaceContainer.push_back(InterafcePort); });
+
+    // so now no one should be available
+    {
+        auto interp = m_portManager->acquireInterfacePortData(iox::capro::Interfaces::INTERNAL, m_Process);
+        EXPECT_EQ(interp, nullptr);
+    }
```

Comment:
this is already checked by `AcquiringOneMoreThanMaximumNumberOfInterfacesFails` and can be removed